thinc/backends/ops.py

```diff
@@ -131,6 +131,15 @@ def _iter_items():
 
         return SizedGenerator(_iter_items, len(sizes))
 
+    # Computed as in "Logistic regression in rare events data"
+    # G King, L Zeng - Political analysis, 2001
+    def balanced_class_weights(self, labels: Ints1d):
```

Comment:
Unused?

Reply:
it is intended to use if the user would like to compute class weights to provide to the `CategoricalCrossEntropy`. But they can give any other kinds of weights, its just there for convenience.

Reply:
Ah, though I wonder if it doesn't better fit in a utility module, since `Ops` is mostly for low-level array operations.

Reply:
Agreed that moving to `loss.py` would make sense!